### PR TITLE
feat(metadata-module): add routing (BS-20)

### PR DESCRIPTION
**Pull request by mpro7**
resolves BS-20

It seems that we have working winner: `svelte-spa-router`.

### CODE REVIEW

File: services/metadata/frontend/content/ProjectPage.svelte [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<style>
+  .container {
+    padding: 40px;
+    display: block;
+    max-width: 1200px;
+  }
+  a {
+    color: var(--dasch-violet);
+  }
+</style>
```
Comment (BalduinLandolt) [inline services/metadata/frontend/content/ProjectPage.svelte:43]: we could argue about the looks of this page ;-)

File: services/metadata/frontend/routes.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import Content from './content/Content.svelte';
+import ProjectPage from './content/ProjectPage.svelte';
+
+export default {
+  '/': Content,
+  '/project/:id': ProjectPage
+}
```
Comment (BalduinLandolt) [inline services/metadata/frontend/routes.js:7]: is there a reason for this to be JavaScript rather than TypeScript?

**Review by BalduinLandolt — APPROVED** [review]
It works nicely!

**Comment by BalduinLandolt** [comment]
Do you want to merge this or do you wait to see how it goes with Sapper and SvelteKit?

File: services/metadata/frontend/content/ProjectPage.svelte (hunk 1 above)
Comment (mpro7) [reply services/metadata/frontend/content/ProjectPage.svelte:43]: What do you mean exactly? This is just placeholder.

File: services/metadata/frontend/routes.js (hunk 2 above)
Comment (mpro7) [reply services/metadata/frontend/routes.js:7]: Good point. I'm not sure if TS is fully supported, will give it a try. Was just following the guide, where JS is used for the routes.

File: services/metadata/frontend/content/ProjectPage.svelte (hunk 1 above)
Comment (BalduinLandolt) [reply services/metadata/frontend/content/ProjectPage.svelte:43]: I know... I was joking.

File: services/metadata/frontend/routes.js (hunk 2 above)
Comment (BalduinLandolt) [reply services/metadata/frontend/routes.js:7]: if it works in TS, that would be more consistent. But having this in JS doesn't hurt either.

**Comment by mpro7** [comment]
> Do you want to merge this or do you wait to see how it goes with Sapper and SvelteKit?

Imo no matters of how it goes with any of above, we need to have some routing here => merge.

**Pull request merged**